spikeinterface/sortingcomponents/motion_estimation.py

```diff
@@ -180,91 +174,132 @@ class DecentralizedRegistration:
     name = 'decentralized'
     params_doc = """
     pairwise_displacement_method: 'conv' or 'phase_cross_correlation'
-        How to estimate the displacement in the parwaise matrix.
+        How to estimate the displacement in the pairwise matrix.
     max_displacement_um: float
-        Maximum possible discplacement in  micrometer.
+        Maximum possible discplacement in micrometers.
     weight_scale: 'linear' or 'exp'
-        For parwaise discplacemtn how to to rescale the associated weight matrix.
+        For parwaise displacement, how to to rescale the associated weight matrix.
     error_sigma: float 0.2
-        In case weight_scale='exp' this control the sigma of the exponential.
+        In case weight_scale='exp' this controls the sigma of the exponential.
     conv_engine: 'numpy' or 'torch'
-        In case of pairwise_displacement_method='conv' what library to use to compute the underlying correlation.
+        In case of pairwise_displacement_method='conv', what library to use to compute
+        the underlying correlation.
     torch_device=None
         In case of conv_engine='torch', you can control which device (cpu or gpu)
     batch_size: int
-        Size of batch for the convolution.
+        Size of batch for the convolution. Increasing this will speed things up dramatically
+        on GPUs and sometimes on CPU as well.
     corr_threshold: float
-        minimum correlation to estimate a motion shift for pairwise displacement bellow the value not used.
+        Minimum correlation between pair of time bins in order for these to be
+        considered when optimizing a global displacment vector to align with
+        the pairwise displacements.
     time_horizon_s: None or float
         When not None the parwise discplament matrix is computed in a small time horizon.
         In short only pair of bins close in time.
         So the pariwaise matrix is super sparse and have values only the diagonal.
-    convergence_method='lsqr_robust' or 'gradient_descent'
+    convergence_method='lsmr', 'lsqr_robust', 'gradient_descent'
```

Comment:
Maybe we could add a little explanation/history of theses methods and related paper no ?

Reply:
Lets keep then a bit more!